config/config.go

```diff
@@ -240,10 +240,21 @@ type Privacy struct {
 	LMT  LMT
 }
 
+type VendorListFetcher struct {
+	// Max concurrent request for downloading the latest version of the vendor list from a specific major version.
```

Comment:
I think that the word "request" should be spelled in the plural form.

Reply:
good catch

Reply:
fixed

Reply:
Thanks !